applications/dashboard/src/scripts/compatibilityStyles/index.ts

```diff
@@ -347,7 +349,7 @@ compatibilityStyles = useThemeCache(() => {
     });
 
     cssOut(`.Item.Read`, {
-        background: "none",
+        backgroundColor: formVars.lists.colors.backgroundColorRead,
```

Comment:
```suggestion
        backgroundColor: colorOut(formVars.lists.colors.backgroundColorRead),
```

After the other changes this should become colorOut instead.